doc/orders.md

```diff
@@ -0,0 +1,194 @@
+# Orders
+
+iOS examples are in Swift, and Android examples are in Kotlin.
+
+- [Fetch Claimed Orders](#fetch-claimed-orders)
+- [Fetch Unclaimed Orders](#fetch-unclaimed-orders)
+- [Observe Orders](#observe-orders)
+- [Claim Orders](#claim-orders)
+- [Update Orders](#update-orders)
+
+## <span id="fetch-claimed-orders">Fetch Claimed Orders</span>
+
+Fetch the latest claimed orders with the server. An order is claimed if it is associated with the current customer.
+
+##### iOS
+
+```swift
+FlyBuy.orders.fetch()
+```
+
+##### Android
+
+```kotlin
+fun fetchOrders() {
+    FlyBuy.orders.fetch { orders, sdkError ->
+        // Handle orders or deal with error
+    }
+}
+```
+
+Return the cached list of orders for the current user.
+
+```
+FlyBuy.orders.all
+```
+
+Return the cached list of open orders for the current user.
+
+```
+FlyBuy.orders.open
+```
+
+Return the cached list of closed orders for the current user.
+
+```
+FlyBuy.orders.closed
+```
+
+## <span id="fetch-unclaimed-orders">Fetch Unclaimed Orders</span>
+
+##### iOS
+
+```swift
+FlyBuy.orders.fetch(withRedemptionCode: code) { (order, error)
+    // Update order claim view with order details here
+}
+```
+
+##### Android
+
+```kotlin
+fun fetchOrder() {
+    FlyBuy.orders.fetch(redemptionCode) { order, sdkError ->
+        // Update order claim view with order details here
+    }
+}
+```
+
+## <span id="observe-orders">Observe Orders</span>
```

Comment:
This section is starting to feel very platform specific. Notifications and observables are slightly different between the tow -- do we want to more this to the separate repos?